### PR TITLE
Index validator fix

### LINKED ISSUE #13: Index validator doesn't work with partitioned tables
Index validator doesn't work with partitioned tables. It is checking only relkind 'r' but for partitioned tables is necessary also check relkind 'p'

### PR DESCRIPTION
**Pull request by petans24**
resolves #13

### CODE REVIEW

File: x/schema/queue.go [hunk 1]
```diff
@@ -27,6 +27,26 @@ func GenerateCreateTableQuery(queueName string) string {
 	`, quotedTableName, quotedTableName[1:len(quotedTableName)-1])
 }
 
+// GenerateCreateTableQueryCompositeIndex returns the query for creating the queue table
+func GenerateCreateTableQueryCompositeIndex(queueName string) string {
```
Comment (prochac) [inline x/schema/queue.go:31]: Could we keep it by the test? This package used to be for testing, but then @kedlas wanted  it to be exported to the world.

**Review by prochac — CHANGES_REQUESTED** [review]
Sorry, this is unfortunate. Are we able to integrate pg_partman in our tests somehow?

**Comment by prochac** [comment]
@petans24 ping, can you hide the func pls? I bet I will forget this till standup 😁

File: x/schema/queue.go (hunk 1 above)
Comment (petans24) [reply x/schema/queue.go]: done

**Review by prochac — APPROVED** [review]

**Pull request merged**